examples/publisher/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "fuel-streams-publisher-examples"
+name = "fuels-example-publisher"
```

Comment:
Sorry, this is a bit confusing. Why this change? Are the examples for `fuels` or `fuel-streams`?